fair/2.0.0/emissions_driven.py

```diff
@@ -97,16 +97,19 @@ def _run_numpy( inp_ar,\
 
     Returns
     -------
-    :obj:`np.ndarray`
-        Results of the run. Given in the format: [[Concentration],[RF],[T],[alpha],[Emissions],[gas parameters],[thermal parameters]]
+    dict
+        Dictionary containing the results of the run. 
+        Keys are 'C', 'RF', 'T', and 'alpha'
+        (Concentration, Radiative Forcing, Temperature and Alpha)
+        Values are in :obj:`np.ndarray` format, with the final index representing 'timestep'
     """
     raise NotImplementedError
     n_species, n_timesteps = inp_ar.shape
-    #C, RF, alpha in format [[species],[time]]
+    #Concentration, Radiative Forcing and Alpha
     C, RF, alpha = np.zeros((3,n_species,n_timesteps))
-    #T in format [time]
+    #Temperature
     T = np.zeros(n_timesteps)
-    #S in format [thermal box]
+    #S represents the results of the calculations from the thermal boxes, an Impulse Response calculation (T = sum(S))
     S = np.zeros_like(d)
     #G_A, G in format [species]
```

Comment:
```suggestion
    #G represents cumulative emissions, while G_A represents emissions accumulated since pre-industrial times, both in the same units as emissions
    # So at any point, G - G_A is equal to the amount of a species that has been absorbed
```